Convert all the data types into a string for data version 0.0.1

yoganandkunche opened the pull request:
### What is the context of this PR?
Converts all data types entered into a string and pushes it to SDX

### How to review 
Launch any survey with dumper
enter different data type values into fields
and verify the dump

Expected: All the values entered must be displayed as strings on the dump

dwyeradam reviewed (changes requested):
This only works for schemas with "data_version": "0.0.1". I'm not sure if this was the intention but if we start to use data version 2 at some point then the conversion will not take place.

ajmaddaford commented:
@dwyeradam That was the intention. The stringifcation of values is only required for surveys that use Q codes.

yoganandkunche committed:
convert all the data types into a string for data version 0.0.1

Previously we used to have int and string data types which where converted into a string before passing on to sdx. Now we have introduced decimals in mwss and now the idea is to convert all the data types which has q_code association to be converted into a string before passing on to SDX.

ajmaddaford reviewed (changes requested):
The commit message needs changing, it's currently `resolved merge conflicts`.

The full commit message should also include why this is being changed.
